documents-storage-jcr/src/main/java/org/exoplatform/documents/storage/jcr/util/JCRDocumentsUtil.java

```diff
@@ -477,6 +481,8 @@ public static void computeDocumentAcl(Node node, AbstractNode documentNode, Iden
           if(identity!=null){
             permissions.add(new PermissionEntry(identity, accessControlEntry.getPermission(),getPermissionRole(accessControlEntry.getMembershipEntry().getMembershipType())));
           }
+        } else if (!membershipEntry.toString().equals(GROUP_ADMINISTRATORS) && groupToIdentity(membershipEntry.getGroup()) != null) {
```

Comment:
Why are you checking on the Administrators group ?

Reply:
@sofyenne could you answer my question here !

Reply:
> Why are you checking on the Administrators group ?

By default, the Administrators group is included as collaborators on the document, so there is no need to display it on the document's collaborators list

Reply:
@ahamdi  No problem if the group administrator is not a collaborator; we check if they are a collaborator in order to not display them on the list .

Reply:
@sofyenne we are not checking wether they are collaborator here, I prefer to remove this check
I don't think there is a requirement to not displaying the administrators grou from the list of collaborators, isn't it ?

Reply:
Done!